cli/src/seth_cli/client/client.go

```diff
@@ -83,7 +83,10 @@ func (c *Client) Get(address []byte) (*EvmEntry, error) {
 	}
 
 	var data string
-	if body.Data != nil {
+	if body.Data == nil {
+		err := fmt.Errorf("Rest API GET received improper response")
+		return nil, err
+	} else {
```

Comment:
Is this still fixing the same problem?

Reply:
yes, its the same problem which you had commented earlier, sorry i messed up git during update of commit, so i created a new PR

Reply:
@knkski yes this issue is fixed, i will close this PR, Thanks